underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -208,12 +221,13 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
         .fixedThreadPoolExecutorServiceFactory("alluxio-s3-transfer-manager-worker",
             numTransferThreads).create();
 
-    TransferManager transferManager = new TransferManager(amazonS3Client, service);
+    TransferManager transferManager = TransferManagerBuilder.standard()
+        .withS3Client(amazonS3Client).withExecutorFactory(() -> service)
+        .withMultipartCopyThreshold(MULTIPART_COPY_THRESHOLD)
+        .build();
```

Comment:
Is this the recommended way of constructing the transfer manager?

Reply:
The old way has been deprecated so I change to use this way which the official doc recommends.